### DISCUSSION TITLE
Various Components Stop Working

### DISCUSSION THREAD

**Discussion by kennygokh** [tadashi-aikawa/obsidian-various-complements-plugin#267] [Q&A]
Hi, when I type a words, it used to be able to display a drop down for me to select the words like note file name etc. However, it just stopped working. I did re-install the plug-in but still face the same issue. I paste my configuration settings below. I would appreciate your advices please.

{
    "version": "9.1.0",
    "mobile": false,
    "settings": {
        "strategy": "default",
        "cedictPath": "./cedict_ts.u8",
        "matchStrategy": "prefix",
        "fuzzyMatch": true,
        "minFuzzyMatchScore": 0.5,
        "matchingWithoutEmoji": true,
        "treatAccentDiacriticsAsAlphabeticCharacters": false,
        "maxNumberOfSuggestions": 5,
        "maxNumberOfWordsAsPhrase": 3,
        "minNumberOfCharactersTriggered": 0,
        "minNumberOfWordsTriggeredPhrase": 1,
        "complementAutomatically": true,
        "delayMilliSeconds": 0,
        "disableSuggestionsDuringImeOn": false,
        "insertAfterCompletion": true,
        "firstCharactersDisableSuggestions": ":/^",
        "patternsToSuppressTrigger": [
            "^~~~.*",
            "^```.*"
        ],
        "phrasePatternsToSuppressTrigger": [],
        "noAutoFocusUntilCycle": false,
        "showMatchStrategy": true,
        "showComplementAutomatically": true,
        "showIndexingStatus": true,
        "descriptionOnSuggestion": "Short",
        "hotkeys": {
            "select": [
                {
                    "modifiers": [],
                    "key": "Enter"
                }
            ],
            "up": [
                {
                    "modifiers": [],
                    "key": "ArrowUp"
                }
            ],
            "down": [
                {
                    "modifiers": [],
                    "key": "ArrowDown"
                }
            ],
            "select 1st": [],
            "select 2nd": [],
            "select 3rd": [],
            "select 4th": [],
            "select 5th": [],
            "select 6th": [],
            "select 7th": [],
            "select 8th": [],
            "select 9th": [],
            "open": [],
            "completion": [],
            "insert as text": []
        },
        "propagateEsc": false,
        "enableCurrentFileComplement": true,
        "currentFileMinNumberOfCharacters": 0,
        "onlyComplementEnglishOnCurrentFileComplement": false,
        "enableCurrentVaultComplement": true,
        "currentVaultMinNumberOfCharacters": 0,
        "includeCurrentVaultPathPrefixPatterns": "",
        "excludeCurrentVaultPathPrefixPatterns": "",
        "includeCurrentVaultOnlyFilesUnderCurrentDirectory": false,
        "enableCustomDictionaryComplement": false,
        "customDictionaryPaths": "https://raw.githubusercontent.com/first20hours/google-10000-english/master/google-10000-english-no-swears.txt",
        "columnDelimiter": "Tab",
        "customDictionaryWordRegexPattern": "",
        "delimiterToHideSuggestion": "",
        "delimiterToDivideSuggestionsForDisplayFromInsertion": "",
        "caretLocationSymbolAfterComplement": "",
        "displayedTextSuffix": " => ...",
        "enableInternalLinkComplement": true,
        "suggestInternalLinkWithAlias": true,
        "excludeInternalLinkPathPrefixPatterns": "",
        "updateInternalLinksOnSave": true,
        "insertAliasTransformedFromDisplayedInternalLink": {
            "enabled": false,
            "beforeRegExp": "",
            "after": ""
        },
        "frontMatterKeyForExclusionInternalLink": "",
        "enableFrontMatterComplement": true,
        "frontMatterComplementMatchStrategy": "inherit",
        "insertCommaAfterFrontMatterCompletion": false,
        "intelligentSuggestionPrioritization": {
            "enabled": true,
            "historyFilePath": "",
            "maxDaysToKeepHistory": 30,
            "maxNumberOfHistoryToKeep": 0
        },
        "disableOnMobile": false,
        "showLogAboutPerformanceInConsole": true,
        "selectionHistoryTree": {}
    }
}

**Comment by tadashi-aikawa** [comment]
Hi, @kennygokh :)

Does that reproduce in the sandbox vault? If so, could you tell me the reproduction steps in the sandbox vault?
